packages/tideline/plugins/blip/basics/chartbasicsfactory.js

```diff
@@ -152,4 +149,6 @@ class BasicsChartNoSize extends React.Component {
   }
 }
 
+
+export { BasicsChartNoSize }
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Why is this exported twice?